misc/experimental/rust/celix_bindings/build.rs

```diff
@@ -44,19 +58,15 @@ fn main() {
     println!("cargo:info=Start build.rs for celix_bindings");
     let include_paths = print_include_paths().unwrap();
 
-    let mut builder = bindgen::Builder::default()
-        .header("src/celix_bindings.h");
+    let mut builder = bindgen::Builder::default().header("src/celix_bindings.h");
 
     // Add framework and utils include paths
     for path in &include_paths {
         builder = builder.clang_arg(format!("-I{}", path));
```

Comment:
As mentioned earlier, `-I` is not enough, we may need `-D` and other compile options (like `-pthread`).
It is hard to tell whether it matters for now without checking each target's `INCLUDE_DIRECTORIES`, `COMPILE_DEFINITIONS` and `COMPILE_OPTIONS`.